js/ui/grid_core/ui.grid_core.columns_controller.js

```diff
@@ -2864,6 +2864,20 @@ module.exports = {
                     }
                     return -1;
                 },
+
+                /**
+                 * @name GridBaseMethods.getVisibleColumnIndex
```

Comment:
Will this method work if: { scrolling: { columnRenderingMode="virtual" } } ?
If so, tests for this should be.